cliquet/__init__.py

```diff
@@ -40,6 +41,7 @@
     'fxa-oauth.scope': 'profile',
     'fxa-oauth.state.ttl_seconds': 3600,  # 1 hour
     'fxa-oauth.webapp.authorized_domains': '',
+    'fxa-oauth.enabled': False,
```

Comment:
It is the webapp client flow only not the full oauth flow.

It will not disable the fact that you can log using a bearer token.

I think we should work on the wording because here it is too ambiguous.